Code/autopkg

```diff
@@ -2683,12 +2683,31 @@ def new_recipe(argv):
                 yaml.dump(recipe, f, encoding="utf-8")
         else:
             with open(filename, "wb") as f:
-                plistlib.dump(recipe, f)
+                plistlib.dump(plist_serializer(recipe), f)
         log(f"Saved new recipe to {filename}")
     except Exception as err:
         log_err(f"Failed to write recipe: {err}")
 
 
+def plist_serializer(obj):
```

Comment:
Can you move this into autopkglib instead and import it into autopkg from autopkglib?

I would like to eventually move as much code as possible out of the autopkg file and put more of it into the library.

Reply:
I actually had this thought for fleeting moment when I was debating where to insert the function...  This makes sense.